core/src/main/java/org/bitcoinj/wallet/DerivationPathFactory.java

```diff
@@ -68,7 +69,23 @@ public ImmutableList<ChildNumber> blockchainIdentityECDSADerivationPath() {
                 .add(ChildNumber.ZERO_HARDENED) //sub feature 0
                 .add(ChildNumber.ZERO_HARDENED) //key type (0 is ECDSA key)
                 .add(ChildNumber.ZERO_HARDENED) //identity index (default to 0 for now)
-                .add(ChildNumber.ZERO_HARDENED) //key index (default to 0 for now)
+                //.add(ChildNumber.ZERO_HARDENED) //key index (default to 0 for now)
```

Comment:
this comment needs to be removed and the documentation above this function is out of date.